Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
Okay thank you!  That worked, but it looks like when I tried to rebase I screwed something up.  I will generate a new pull request with the listed changes.